packages/react-pdf/src/Page/AnnotationLayer.tsx

```diff
@@ -158,7 +158,16 @@ export default function AnnotationLayer() {
 
     const clonedViewport = viewport.clone({ dontFlip: true });
 
-    const parameters = {
+    const annotationLayerParameters = {
+      accessibilityManager: null, // TODO: Implement this
+      annotationCanvasMap: null, // TODO: Implement this
+      div: layer,
+      l10n: null, // TODO: Implement this
+      page,
+      viewport: clonedViewport,
+    };
```

Comment:
Need to understand the impact of these TODOs

Reply:
Looks like no regressions; only some new functionalities won't get enabled. Not a show stopper.